kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ActiveAddFilesIterator.java

```diff
@@ -103,33 +112,37 @@ public void close() throws IOException {
 
     /**
      * Grabs the next FileDataReadResult from `iter` and updates the value of `next`.
-     *
+     * <p>
      * Internally, implements the following algorithm:
      * 1. read all the RemoveFiles in the next ColumnarBatch to update the `tombstonesFromJson` set
      * 2. read all the AddFiles in that same ColumnarBatch, unselecting ones that have already
-     *    been removed or returned by updating a selection vector
+     * been removed or returned by updating a selection vector
      * 3. produces a DataReadResult by dropping that RemoveFile column from the ColumnarBatch and
-     *    using that selection vector
-     *
+     * using that selection vector
+     * <p>
      * Note that, according to the Delta protocol, "a valid [Delta] version is restricted to contain
      * at most one file action of the same type (i.e. add/remove) for any one combination of path
      * and dvId". This means that step 2 could actually come before 1 - there's no temporal
      * dependency between them.
-     *
+     * <p>
      * Ensures that
      * - `next` is non-empty if there is a next result
      * - `next` is empty if there is no next result
      */
     private void prepareNext() {
-        if (next.isPresent()) return; // already have a next result
-        if (!iter.hasNext()) return; // no next result, and no batches to read
+        if (next.isPresent()) {
+            return; // already have a next result
+        }
+        if (!iter.hasNext()) {
+            return; // no next result, and no batches to read
+        }
 
         final Tuple2<FileDataReadResult, Boolean> _next = iter.next();
         final FileDataReadResult fileDataReadResult = _next._1;
         final boolean isFromCheckpoint = _next._2;
         final ColumnarBatch addRemoveColumnarBatch = fileDataReadResult.getData();
 
-        assert(addRemoveColumnarBatch.getSchema().equals(LogReplay.ADD_REMOVE_READ_SCHEMA));
+        assert (addRemoveColumnarBatch.getSchema().equals(LogReplay.ADD_REMOVE_READ_SCHEMA));
```

Comment:
don't asserts do nothing by default? use checkArgument?

Reply:
depends if we want this to run in production code or only during test execution. what do you all think?